### ISSUE TITLE
Progress doesn't get shown

### ISSUE THREAD

**Issue by z3ntu** [Microsoft/azure-vhd-utils-for-go#16]
The console only shows

     Completed:   0% [      0.00 MB] RemainingTime: 00h:12m:43s Throughput: 0 MB/sec   | 

, but apparently it is still uploading.

**Comment by anuchandy** [comment]
@z3ntu was your upload bandwidth is less than 1 MB? If its in Kbs then it's gets round down to 0 when converted to Mbs. It's an easy fix.

Between were you able to upload the VHD successfully ?

**Comment by z3ntu** [comment]
Yes, my upload bandwidth is unfortunately less than 1MBit/s (768kbit/s according to the isp) and yes it seems that it was uploaded successfully (couldn't verify as the vm didn't start then, but that could be because I installed the grub bootloader for uefi and not bios).

**Comment by anuchandy** [comment]
@z3ntu thanks for confirming.
